source/getting_started/aiot_intelligent_edge/architecture_aiot.rst

```diff
@@ -3,6 +3,7 @@
 Architecture
 ============
 
-<Content here - Owner: Ashish
-Suggestion from Ashish - I have all those visuals, so I will put this together along with text to help illustrate the Control Plane, Data Plane, the various layers in the stack, how we replace MQTT for telemetry data, etc.>
+..
```

Comment:
What did you change here?

Reply:
turned the content in angular braces to a multi line comment.